pkg/operator/ceph/cluster/watcher.go

```diff
@@ -598,13 +598,13 @@ func (c *clientCluster) unfenceAndDeleteNetworkFence(ctx context.Context, node c
 		return err
 	}
 
-	err = wait.PollUntilContextTimeout(ctx, 2*time.Second, 60*time.Second, true, func(ctx context.Context) (bool, error) {
+	err = wait.PollUntilContextTimeout(ctx, 2*time.Second, 120*time.Second, true, func(ctx context.Context) (bool, error) {
 		err = c.client.Get(ctx, types.NamespacedName{Name: fenceResourceName(node.Name, driver, cluster.Namespace)}, networkFence)
 		if err != nil && !errors.IsNotFound(err) {
 			return false, err
 		}
 
-		if networkFence.Spec.FenceState != addonsv1alpha1.Unfenced {
+		if networkFence.Spec.FenceState != addonsv1alpha1.Unfenced || addonsv1alpha1.FenceState(networkFence.Status.Result) != addonsv1alpha1.FenceState(addonsv1alpha1.FencingOperationResultSucceeded) {
```

Comment:
```suggestion
check for `csiaddonsv1alpha1.UnFenceOperationSuccessfulMessage` message , Result might contains older details or we need to have a Result not a generic one from the networkfence CR
```

`if networkFence.Spec.FenceState != addonsv1alpha1.Unfenced` IMHO this check is redundunt here

Reply:
okay, I have removed `if networkFence.Spec.FenceState != addonsv1alpha1.Unfenced` and added check for `status.Message`